libs/sdk-ui-dashboard/src/model/queries/connectingAttributes.ts

```diff
@@ -0,0 +1,38 @@
+// (C) 2022 GoodData Corporation
+
+import { ObjRef } from "@gooddata/sdk-model";
+import { IDashboardQuery } from "./base";
+
+/**
+ * @internal
+ */
+export interface QueryConnectingAttributes extends IDashboardQuery {
+    type: "GDC.DASH/QUERY.CONNECTING.ATTRIBUTES";
+    payload: {
+        readonly refs: ObjRef[][];
```

Comment:
Should this be 
```ts
readonly refs: [ObjRef, ObjRef][]
```
? So it enforces pairs?

Reply:
type updated